### PR TITLE
Human survey irb changes

### PR DESCRIPTION
**Pull request by squirrelo**
This adds and renames questions to get the survey in line with the new survey changes in the IRB.

### CODE REVIEW

File: amgut/db/patches/0032.sql [hunk 1]
```diff
@@ -0,0 +1,369 @@
...
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Zimbabwe', 244);
+
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES
+(149, 'Unspecified', 0), (149, 'Yes', 1), (149, 'No', 2);
+
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES
+(152, 'Unspecified', 0),
+(152, 'I tend to be constipated (have difficulty passing stool) - Type 1 and 2', 1),
+(152, 'I tend to have diarrhea (watery stool) - Type 5, 6 and 7', 2),
+(152, 'I tend to have normal formed stool - Type 3 and 4', 3);
```
Comment (mortonjt) [inline amgut/db/patches/0032.sql]: Question - would it be more desirable to separate out the types, or leave it as is?

File: amgut/db/patches/0032.sql [hunk 2]
```diff
@@ -0,0 +1,369 @@
...
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Spain', 204);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Sri Lanka', 205);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Sudan', 206);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Suriname', 207);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Svalbard and Jan Mayen', 208);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Swaziland', 209);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Sweden', 210);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Switzerland', 211);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Syrian Arab Republic', 212);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Taiwan, Province of China', 213);
```
Comment (mortonjt) [inline amgut/db/patches/0032.sql]: Hmm, some people might get upset about this.

File: amgut/db/patches/0032.sql (hunk 1 above)
Comment (squirrelo) [inline amgut/db/patches/0032.sql]: We can't change this: This is what was submitted to IRB

File: amgut/db/patches/0032.sql (hunk 2 above)
Comment (squirrelo) [inline amgut/db/patches/0032.sql]: That's what it is in all other questions. Not sure if we want to fix this or not.

File: amgut/db/patches/0032.sql [hunk 3]
```diff
@@ -0,0 +1,369 @@
...
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Jordan', 112);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Kazakhstan', 113);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Kenya', 114);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Kiribati', 115);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Korea, Democratic People''s Republic of', 116);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Korea, Republic of', 117);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Kuwait', 118);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Kyrgyzstan', 119);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Lao People''s Democratic Republic', 120);
```
Comment (mortonjt) [inline amgut/db/patches/0032.sql]: I'm not familiar enough with postgres.  Is this the is `''` the proper way to do escape characters?

File: amgut/db/patches/0032.sql [hunk 4]
```diff
@@ -0,0 +1,369 @@
...
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Cayman Islands', 41);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Central African Republic', 42);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Chad', 43);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Chile', 44);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'China', 45);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Christmas Island', 46);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Cocos (Keeling) Islands', 47);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Colombia', 48);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Comoros', 49);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Congo', 50);
```
Comment (mortonjt) [inline amgut/db/patches/0032.sql]: There are two congos?

Comment (squirrelo) [inline amgut/db/patches/0032.sql]: yes

File: amgut/db/patches/0032.sql (hunk 3 above)
Comment (squirrelo) [inline amgut/db/patches/0032.sql]: Yup, tests pass so it must be :)

File: amgut/db/patches/0032.sql (hunk 1 above)
Comment (mortonjt) [inline amgut/db/patches/0032.sql]: Fair enough.

File: amgut/db/patches/0032.sql [hunk 5]
```diff
@@ -0,0 +1,369 @@
...
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'United Arab Emirates', 229);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'United Kingdom', 230);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'United States', 231);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'United States Minor Outlying Islands', 232);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Uruguay', 233);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Uzbekistan', 234);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Vanuatu', 235);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Venezuela', 236);
+INSERT INTO survey_question_response (survey_question_id, response, display_index) VALUES (148, 'Viet Nam', 237);
```
Comment (mortonjt) [inline amgut/db/patches/0032.sql]: Viet Nam?

**Comment by mortonjt** [comment]
Tested out the survey - so far so good.

A few concerns with the countries - could you provide a link for where you obtained the country names?

**Comment by squirrelo** [comment]
This was copied from the original set up, so I unfortunately have no idea where that came from.

File: amgut/db/patches/0032.sql (hunk 3 above)
Comment (mortonjt) [inline amgut/db/patches/0032.sql]: Awesome.  Looks like that is rendering properly :)

**Comment by squirrelo** [comment]
Changes made, ready to go

**Comment by mortonjt** [comment]
Alright.  Looking good to me.  Is this ready to merge?

**Pull request merged**